pkg/rpc/client.go

```diff
@@ -128,3 +124,92 @@ func (c *Client) rpcRequest(ctx context.Context, data io.Reader) ([]byte, error)
 
 	return body, nil
 }
+
+func (c *Client) getResponse(ctx context.Context, method string, params []interface{}, result HasRPCError) error {
+	body, err := c.rpcRequest(ctx, formatRPCRequest(method, params))
+	// check if there was an error making the request:
+	if err != nil {
+		return fmt.Errorf("%s RPC call failed: %w", method, err)
+	}
+	// log response:
+	klog.V(2).Infof("%s response: %v", method, string(body))
```

Comment:
Can this be debug please? This could be a large volume potentially.

Reply:
Klog doesn't seem to have a `Debugf` method, my impression was that the `V(2)` is the klog equivalent of debug lgos

Reply:
It is such garbage 😢 

I'm going to file a ticket to replace it.